localisation/00_wc_vanilla_replacements.csv

```diff
@@ -153,4 +153,33 @@ HF_222_DESC;My servants apparently lost track of [child_of_destiny.GetFirstName]
 HF_222_OPTA;[child_of_destiny.GetFirstName], oh how you scared me...! And how do you know Kalimag?;;[child_of_destiny.GetFirstName], oh wie habt Ihr mich erschreckt...! Und woher kennt Ihr Kalimag?;;;;;;;;;;;x
 child_of_mithridates;Child of Shadra;;Kind von Shadra;;;;;;;;;;;x
 child_of_mithridates_desc;This character resists poison to such a degree that people claim [GetHerHim] a child of Shadra.;;Dieser Charakter widersteht Gift in einem solchen Ma�e, dass die Leute behaupten, [GetSheHe] sei ein Kind von Shadra.;;;;;;;;;;;x
-bloodline_desc_child_of_destiny;A [child_of_destiny.GetManWoman] of true legend. [child_of_destiny.GetHerHisCap] exploits mimicked those of the ancient heroes!;;Ein wahres [child_of_destiny.GetManWoman] der Legenden. [child_of_destiny.GetSheHeCap] eifert den Heldentaten der alten Helden nach!;;;;;;;;;;;x
\ No newline at end of file
+bloodline_desc_child_of_destiny;A [child_of_destiny.GetManWoman] of true legend. [child_of_destiny.GetHerHisCap] exploits mimicked those of the ancient heroes!;;Ein wahres [child_of_destiny.GetManWoman] der Legenden. [child_of_destiny.GetSheHeCap] eifert den Heldentaten der alten Helden nach!;;;;;;;;;;;x
```

Comment:
```suggestion
bloodline_desc_child_of_destiny;A [child_of_destiny.GetManWoman] of true legend. [child_of_destiny.GetHerHisCap] exploits mimicked those of ancient heroes!;;Ein wahres [child_of_destiny.GetManWoman] der Legenden. [child_of_destiny.GetSheHeCap] eifert den Heldentaten der alten Helden nach!;;;;;;;;;;;x
```

Reply:
It is not part of this PR
@zumbak04. @arithon, can you apply this suggestion it in the next localization branch, please?

Reply:
he mentioned it in a couple PRs recently... did anyone change it anywhere?

Reply:
@DukeJeans @MemeWizard @arithon I already changed it on the bug crusade.